.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "i18n-ally.localesPaths": ["src/translations/locales"]
```

Comment:
ooh this is interesting, is there an extension that hooks into this?

Reply:
sent this to you already!